leda/components/AutoComplete/types.ts

```diff
@@ -12,12 +12,13 @@ import type { COMPONENTS_NAMESPACES } from '../../constants';
  * The property's name is passed through textField
 */
 export interface DataObject {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why can't we specify real type here?

Reply:
because the value can be literaly any
we can use unknown, but it requires code changes in many other places, it is not necessary